test/claims-status/components/appeals-v2/Timeline.unit.spec.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
+
+import Timeline from '../../../../src/js/claims-status/components/appeals-v2/Timeline';
+
+import { getEventContent } from '../../../../src/js/claims-status/utils/appeals-v2-helpers';
+
+const defaultProps = {
+  events: [
+    {
+      type: 'claim',
+      date: '2016-05-30',
+      details: {}
+    },
+    {
+      type: 'nod',
+      date: '2016-06-10',
+      details: {}
+    }
+  ],
+  currentStatus: {
+    title: 'Current Status Title',
+    description: 'Status description here'
+  }
+};
+
+describe('<Timeline/>', () => {
+  it('should render', () => {
+    const wrapper = shallow(<Timeline {...defaultProps}/>);
+    expect(wrapper.type()).to.equal('ol');
+  });
+
+  it('should expand and collapse past history events', () => {
+    const component = shallow(<Timeline {...defaultProps}/>);
+    const expander = component.find('.section-unexpanded button');
+    expect(expander.exists()).to.be.true;
```

Comment:
Not sure if this works - I believe .find() will always return a`ShallowWrapper` even if it doesn't find a matching node, so `exists()` might always return `true`. From the enzyme docs, it seems as though `expect(wrapper.find('.some-class').length).to.equal(1)` is the idiomatic way of doing it...which I'm on the fence about since it kind of obscures what's being tested.

Reply:
I just tested this by changing the `.find()` selector to something that doesn't exist (`.section-unexpanded buttons` with an s) and the test failed, so I think we're golden. :crossed_fingers: